cmd/create/cluster/cmd.go

```diff
@@ -230,12 +230,11 @@ func init() {
 			"Leave empty to use an auto-generated one.",
 	)
 	flags.StringVar(
-		&args.operatorRolesPath,
-		"operator-roles-path",
+		&args.operatorRolePath,
+		"operator-role-path",
```

Comment:
do we need to deprecate the previous name of the flag? add an alias for the old one? update the docs?

Reply:
it was never released. we are still working on the release.

Reply:
> We need to add `flags.MarkHidden("path")` to `cmd/create/userrole/cmd.go` and `cmd.Flags().Changed("path")` to the interactive mode.
> 
> Left a couple of comments, except that LGTM.

done